.coverage

```diff
@@ -1 +1 @@
-!coverage.py: This is a private format, don't read it directly!{"lines":{"/home/salma/Desktop/Politicosystem/app/__init__.py":[1,2,3,4,7,18,29,40,43,49,50,51,52,53,54],"/home/salma/Desktop/Politicosystem/app/api/__init__.py":[1],"/home/salma/Desktop/Politicosystem/app/api/v1/__init__.py":[1],"/home/salma/Desktop/Politicosystem/app/api/v1/views/__init__.py":[1],"/home/salma/Desktop/Politicosystem/app/api/v1/views/officeView.py":[2,3,4,5,7,10,53,61,13,14,15,16,18,22,26,31,44,45,47,48,49,50,32,33,34,35,27,28,29,30,19,20,21,23,25,56,57,58,64,66,68,69,71,72,73,74,37,39,40,41,42],"/home/salma/Desktop/Politicosystem/app/api/v1/models/__init__.py":[1],"/home/salma/Desktop/Politicosystem/app/api/v1/models/officemodel.py":[1,4,5,7,13,23,29,10,11,17,18,19,21,27,34,35],"/home/salma/Desktop/Politicosystem/app/api/v1/views/partyView.py":[1,2,3,4,6,9,60,68,85,98,12,13,15,16,17,19,23,27,31,37,50,51,52,54,55,56,57,38,39,40,41,43,45,46,47,48,24,26,28,30,20,21,22,32,33,34,35,88,89,93,94,95,90,91,92,63,64,65,72,74,76,77,79,80,81,82,101,102,103,104,105,106,107],"/home/salma/Desktop/Politicosystem/app/api/v1/models/partyModel.py":[1,4,5,7,14,24,30,36,46,10,11,12,17,18,19,20,22,39,40,41,44,42,43,28,34,50,51,52,53,54,55]}}
```

Comment:
Kindly add this `.coverage` file in the `.gitignore` so that it may not be committed. Otherwise good job here.